es/lecciones/escritura-sostenible-usando-pandoc-y-markdown.md

```diff
@@ -310,10 +322,10 @@ csl: mla.csl
 ---
 ```
 
-utiliza entonces el mismo comando:
+Después simplemente utiliza el encantamiento de pandoc para lanzar tu archivo de markdown en tu formato objetivo (.pdf o .docx):
```

Comment:
encantamiento?

Reply:
Supongo que se refiere a “la magia” de Pandoc porque convierte el markdown en otro formato? Si es asi, lo borraría (simplemente utiliza Pandoc) o lo sustituiría por algo más formal como “la funcionalidad” al ser un contexto de publicación

Reply:
ah, entiendo... fuera de contexto, la verdad es que no lo entendía...

Reply:
mmm, estaba volviendo a ver cómo está en inglés. Aparece así: 
"However, we prefer to use the YAML header options whenever possible, since it makes our command line incantations easier to type and to remember." 
"You then repeat the pandoc incantation to cast your markdown file into your target format."

Reply:
Ya veo. Literal lo pone como si se tratara de una clase de Harry Potter. La incantation es el hechizo (las palabras) y to cast es su lanzamiento (?). Si queremos mantener ese vocabulario propongo utilizar hechizo por encantamiento y transformar por lanzar, además de tu formato seleccionado (en vez de objetivo):
Después simplemente utiliza el hechizo de Pandoc para transformar tu archivo en markdown al formato seleccionado (.pdf o .docx)

¿Qué os parece? Si nos deshacemos de la magia, podemos poner algo como “comando”.

Reply:
Me parece bien. Voy a hacer la modificación y esperemos a ver si @vgayolrs quien fue el traductor de este tutorial quiere modificar algo

Reply:
Aunque creo que mejor quitar el lenguaje Harry Potter?